cardano-db/src/Cardano/Db/Migration/Version.hs

```diff
@@ -40,18 +40,18 @@ nextMigrationVersion (MigrationVersion _stage ver _date) = do
   (y, m, d) <- Time.toGregorian . Time.utctDay <$> Time.getCurrentTime
   pure $ MigrationVersion 2 (ver + 1) (fromIntegral y * 10000 + m * 100 + d)
 
-renderMigrationVersion :: MigrationVersion -> String
+renderMigrationVersion :: MigrationVersion -> Text
 renderMigrationVersion mv =
-  List.intercalate "-"
+  toS $ intercalate "-"
```

Comment:
Why are we moving from qualified ` List.intercalate` to unqualified `intercalate` ? I know `Cardano.Prelude` exposes `intercalate` but I do not see a good reason to switch from qualified to unqualified usage.

Reply:
https://github.com/input-output-hk/cardano-db-sync/pull/527#discussion_r575045248